src/Components/SavedSearchAlert/SavedSearchAlertModal.tsx

```diff
@@ -222,17 +224,40 @@ export const SavedSearchAlertModal: FC<SavedSearchAlertFormProps> = ({
               <SavedSearchAlertNameInputQueryRenderer />
               <Box>
                 <Text variant="xs">Filters</Text>
-                <Spacer y={2} />
+                <Spacer y={1} />
                 <Flex flexWrap="wrap" gap={1}>
                   <SavedSearchAlertPills
                     items={pills}
                     onDeletePress={handleRemovePillPress}
                   />
                 </Flex>
 
-                <Separator my={1} />
+                <Separator my={2} />
+
+                <Media at="xs">
+                  <PriceRangeFilter expanded={false} />
+                </Media>
+                <Media greaterThan="xs">
+                  <PriceRangeFilter expanded={true} />
+                </Media>
```

Comment:
**suggestion**: I remember discussing this with Barney and we can keep this `PriceRangeFilter` collapsed in either case. We'll soon replace it with the new filters CTA.